lib/admin_credentials.rb

```diff
@@ -19,7 +19,21 @@ def initialize(env, **opts)
 
     # @return [APIAccessor]
     def get
-      res = master_host.exec_as(:admin, "cat /root/.kube/config", quiet: true)
+      res = master_host.exec_admin("cat /root/.kube/config", quiet: true)
```

Comment:
Hi @akostadinov , tried `ocp4` in jobs [37492/consoleFull](https://openshift-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/Runner-v3/37492/consoleFull), 37493/console , both failed with error:
```
    When I run the :annotate admin command with:        # features/step_definitions/cli.rb:31
...
      Inappropriate ioctl for device (Errno::ENOTTY)
      /home/jenkins/workspace/Runner-v3/lib/ssh.rb:244:in `initialize'
      /home/jenkins/workspace/Runner-v3/lib/host.rb:744:in `new'
...
      /home/jenkins/workspace/Runner-v3/lib/admin_credentials.rb:22:in `get'
```
Env is good because then I turned back to my former workaround as below and that still passes ([37495/console](https://openshift-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/Runner-v3/37495/console)):
```
[root@ip-10-0-11-47 ~]# mkdir -p ~/.ssh; chmod 700 ~/.ssh; cat /home/core/.ssh/authorized_keys >> ~/.ssh/authorized_keys; mkdir ~/.kube; mv `find /etc/kubernetes/static-pod-resources/ -name "kubeconfig" | head -n 1` ~/.kube/
```

Reply:
In console log I see `core@ec2-18-191-132-235.us-east-2.compute.amazonaws.com's password:      # Update with cluster-admin (*admin* command)`. This looks like SSH login to machine failed. Which is unrelated to this PR.

You sure installer created the machine with the correct key?

Reply:
Sure correct key, compare below:
```
[tester@fedora28 ~]$ rm -rf ~/.ssh
[tester@fedora28 ~]$ ssh core@ec2-...-235.us-east-2.compute.amazonaws.com
Are you sure you want to continue connecting (yes/no)? yes
core@ec2-...-235.us-east-2.compute.amazonaws.com's password:

[tester@fedora28 ~]$ ssh -i ./libra.pem core@ec2-...-235.us-east-2.compute.amazonaws.com
Last login: Tue Jan  8 08:25:03 2019 from ...
...
[core@ip-10-0-11-47 ~]$
```
I asked other colleague @yanpzhan trying their subteam env and met same error [37496/console](https://openshift-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/Runner-v3/37496/console)

Reply:
In my cluster it has been working fine. I've never did your workaround - neither with `root` nor kubeconfig location. Tomorrow I'll create a new cluster and see if something changed.

Reply:
I managed to reproduce.

Reply:
missing private config for `ocp4`. Sorry about that. This is why it worked locally but not for you. Updating your private repos sohuld make it work for you locally. In Jenkins it should already work.

Reply:
@akostadinov  I tried on Mac, but met this error

>     Given I open admin console in a browser                        # features/step_definitions/admin_console.rb:6
      [01:46:46] INFO> Remote cmd: `mkdir -v -p '/tmp/workdir/Yadans-MacBook-Pro-yapei'` @ssh://core@ec2-18-216-125-168.us-east-2.compute.amazonaws.com
      /etc/profile.d/lang.sh: line 19: warning: setlocale: LC_CTYPE: cannot change locale (UTF-8): No such file or directory
      mkdir: created directory '/tmp/workdir'
      mkdir: created directory '/tmp/workdir/Yadans-MacBook-Pro-yapei'
      
      [01:46:47] INFO> Exit Status: 0
      [01:46:52] ERROR> /etc/profile.d/lang.sh: line 19: warning: setlocale: LC_CTYPE: cannot change locale (UTF-8): No such file or directory
      cat: /etc/profile.d/lang.sh: line 19: warning: setlocale: LC_CTYPE: cannot change locale (UTF-8): No such file or directory: No such file or directory
      
      error getting kubeconfig from master ec2-18-216-125-168.us-east-2.compute.amazonaws.com, see log (RuntimeError)
      /Users/yapei/repo/verification-tests/lib/admin_credentials.rb:60:in `get'
      /Users/yapei/repo/verification-tests/lib/environment.rb:89:in `admin'
      /Users/yapei/repo/verification-tests/lib/world.rb:100:in `admin'
      /Users/yapei/repo/verification-tests/features/step_definitions/admin_console.rb:3:in `/^default admin-console route is stored in the(?: :(\S+))? clipboard$/'
      /Users/yapei/repo/verification-tests/features/step_definitions/admin_console.rb:11:in `/^I open admin console in a browser$/'
      features/tierN/web/admin-console/build.feature:12:in `Given I open admin console in a browser'

The errors shows ``

Reply:
for Mac users, you have to set some variables to make LC_CTYPE disappear.
```
export LC_ALL=en_US.UTF-8
export LANG=en_US.UTF-8
```
And better make this in your ~/.bashrc.

Reply:
Thanks @wjiangjay  Adding these fix the problem, but I didn't have this setting for 3.x auto tests and they work.

Reply:
@wjiangjay , we can change `conf = YAML.load(res[:response])` to `conf = YAML.load(res[:stdout])` in lib/admin_credentials.rb:46 so that error messages are ignored.

Reply:
make sense!